assets/js/handlebars/templates/settings-modal-body.handlebars

```diff
@@ -0,0 +1,24 @@
+<div id="settings-modal-content">
+  <label class="switch-button-container">
+    <span>Night mode</span>
+    <div class="switch-button">
+      <input class="switch-button__checkbox" type="checkbox" name="night_mode" />
+      <div class="switch-button__bg"></div>
+    </div>
+  </label>
+  <label class="switch-button-container">
+    <span>Tooltips</span>
```

Comment:
```suggestion
    <span>Show tooltips on mouse over</span>
```

?

Reply:
Doesn't tooltip imply that it's on mouse over though? Maybe we should clarify the context, like "Show tooltips on code references"?

Reply:
```suggestion
    <span>Show tooltips on code references</span>
```